protected/humhub/modules/admin/models/forms/FileSettingsForm.php

```diff
@@ -30,7 +30,7 @@ public function init()
         $this->maxFileSize = $settingsManager->get('maxFileSize') / 1024 / 1024;
         $this->maxPreviewImageWidth = $settingsManager->get('maxPreviewImageWidth');
         $this->maxPreviewImageHeight = $settingsManager->get('maxPreviewImageHeight');
-        $this->hideImageFileInfo = $settingsManager->get('hideImageFileInfo');
+        $this->excludeMediaFilesPreview = $settingsManager->get('excludeMediaFilesPreview');
```

Comment:
Maybe we should return a default of 1 here (second parameter), since the setting won't be available in existing installations.

Reply:
No, actually this default value in `FileSettingForm` is just used in rendering the file setting form and it's confusing for users. because they see a checked checkbox for this setting but in practice, it's not.

Reply:
You are right, if doing so there should be a central getter function for this setting. My concern is that the default will be null here which is not the behavior we aim for, the default sould be '1', but this won't be the case for existing installations. Maybe we should add a migration to use the old setting value.